.github/ISSUE_TEMPLATE/formatting.md

```diff
@@ -47,3 +47,5 @@ Don't fill the form below manually! Let a program create a report for you:
 ```
 
 **Expected behavior:**
+
+<!-- Please write both text and code -->
```

Comment:
Don't know, but feel hard to understand "text" and "code". Maybe @kachkaev or someone else have a better idea?

Reminder: Please sync this https://github.com/prettier/prettier/blob/e9b8764fc4c770aaa8e5d9b982bda40a83a4cf2a/website/playground/markdown.js#L30 before merge.

Reply:
Thank you for pointing it out. I would like to know if there is a better wording.

Reply:
How about something like this?

```suggestion
<!-- Please insert a code snippet and its description -->
```

```suggestion
<!-- Please insert a code snippet and supplement it with a short description -->
```

Reply:
Maybe we should add an "Expected output" section?

Reply:
````markdown
**Output:**

```jsx
// code snippet
```

**Expected output:**

```jsx
// code snippet
```

**Why?**

<!-- Short explanation -->
````

Reply:
looks good to me